sbor/Cargo.toml

```diff
@@ -5,15 +5,19 @@ edition = "2021"
 
 [dependencies]
 sbor-derive = { path = "../sbor-derive" }
-hashbrown = { version = "0.12.1", optional = true }
+hashbrown = { version = "0.13.2", optional = true } # Align with the version indexmap pulls in to reduce the dependency tree
```

Comment:
This comment is slightly misleading, as cargo select versions to de-duplicate.

Reply:
Based on the simulator's lock file, it seemed to do that with patch versions, but not with minor versions -- but maybe I just needed to delete it and recreate it 🤷